mmv1/third_party/terraform/tests/resource_gke_hub_feature_membership_test.go.erb

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
-  gkehub "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/gkehub/beta"
+  gkehub "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/gkehub/<% if version == 'beta' -%>beta<% else -%>alpha<% end -%>"
```

Comment:
Ah- we've gotta do this manually because the DCL calls it alpha but we call it private, right? Can we add a function so that we can write it as the following?

```suggestion
  gkehub "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/gkehub/<%= dcl_version(version) -%>"
```